lib/runtime-core/src/vmcalls.rs

```diff
@@ -31,16 +43,32 @@ pub unsafe extern "C" fn local_static_memory_grow(
     ret
 }
 
+/// Get the size in [`Pages`] of a local [`StaticMemory`].
```

Comment:
Optional: I'd rather that were
```suggestion
/// Get the size of a local [`StaticMemory`] in [`Pages`].
```